README.md

```diff
@@ -38,6 +38,9 @@ Basic models are:
 * `Wish` is Group User relation, ie current location
```

Comment:
Hi @DeusCorona ,

## Wow, you did it 🎉 
![Brilliant](https://media.giphy.com/media/d31w24psGYeekCZy/giphy.gif)



### To highlight:

- Nice devise-forms style✔️ 
- Nice navbar style✔️ 
- Nice footer style ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Reply:
Thank you sir! Hoping to improve it even further.

Reply:
Sure, we will improve it together.